--- previous
+++ current
@@ -1,7 +1,6 @@
 <div id="header">
  <h1>
   <span class="image">
-       <!-- test -->
    <img alt="./images/proxmox-logo.svg" src="data:image/svg+xml;base64,
 PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+Cjwh
 LS0gQ3JlYXRlZCB3aXRoIElua3NjYXBlIChodHRwOi8vd3d3Lmlua3NjYXBlLm9yZy8pIC0tPgoK
@@ -784,7 +783,6 @@
       <span class="monospaced">
        vzdump
       </span>
-      <!-- test -->
       ) creates consistent snapshots of
 running containers and KVM guests. It creates an archive of the VM or
 container data, including the configuration files.
